runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -0,0 +1,2505 @@
+				assert_eq!(
+					pallet_balances::Pallet::<T>::free_balance(&parachain_account(
+						T::FudgeHandle::SIBLING_ID
+					)),
+					transfer_amount
+				);
+			});
+
+			env.pass(Blocks::ByNumber(1));
+
+			env.sibling_state_mut(|| {
+				let current_balance =
+					orml_tokens::Pallet::<T>::free_balance(air_in_sibling, &Keyring::Bob.into());
+
+				// Verify that Keyring::Bob now has (amount transferred - fee)
+				assert_eq!(current_balance, transfer_amount - fee(18));
+
+				// Sanity check for the actual amount Keyring::Bob ends up with
+				assert_eq!(current_balance, 4992960800000000000);
+			});
+		}
+
+		fn test_air_transfers_to_and_from_sibling<T: Runtime + FudgeSupport>() {
+			let mut env = FudgeEnv::<T>::from_storage(
+				Genesis::default()
+					.add(genesis::balances::<T>(air(10)))
+					.storage(),
+				Default::default(),
+			);
+
+			setup_xcm(&mut env);
+
+			// In order to be able to transfer AIR from Sibling to Altair, we need to first
+			// send AIR from Altair to Sibling, or else it fails since it'd be like Sibling
+			// had minted AIR on their side.
+			transfer_air_to_sibling(&mut env);
+
+			let alice_initial_balance = air(5);
+			let bob_initial_balance = air(5) - air_fee();
+			let transfer_amount = air(1);
+
+			// Note: This asset was registered in `transfer_air_to_sibling`
+			let air_in_sibling = CurrencyId::ForeignAsset(12);
+
+			env.parachain_state_mut(|| {
+				assert_eq!(
+					pallet_balances::Pallet::<T>::free_balance(&Keyring::Alice.into()),
+					alice_initial_balance
+				);
+			});
+
+			env.sibling_state_mut(|| {
+				assert_eq!(
+					pallet_balances::Pallet::<T>::free_balance(&parachain_account(
+						T::FudgeHandle::PARA_ID
+					)),
+					0
+				);
+				assert_eq!(
+					orml_tokens::Pallet::<T>::free_balance(air_in_sibling, &Keyring::Bob.into()),
+					bob_initial_balance
+				);
+
+				assert_ok!(orml_xtokens::Pallet::<T>::transfer(
+					RawOrigin::Signed(Keyring::Bob.into()).into(),
+					air_in_sibling,
+					transfer_amount,
+					Box::new(
+						MultiLocation::new(
+							1,
+							X2(
+								Parachain(T::FudgeHandle::PARA_ID),
+								Junction::AccountId32 {
+									network: None,
+									id: Keyring::Alice.into(),
+								}
+							)
+						)
+						.into()
+					),
+					WeightLimit::Limited(8_000_000_000_000.into()),
+				));
+
+				// Confirm that Bobs's balance is initial balance - amount transferred
+				assert_eq!(
+					orml_tokens::Pallet::<T>::free_balance(air_in_sibling, &Keyring::Bob.into()),
+					bob_initial_balance - transfer_amount
+				);
+			});
+
+			env.pass(Blocks::ByNumber(2));
+
+			env.parachain_state_mut(|| {
+				// Verify that Keyring::Alice now has initial balance + amount transferred - fee
+				assert_eq!(
+					pallet_balances::Pallet::<T>::free_balance(&Keyring::Alice.into()),
+					alice_initial_balance + transfer_amount - air_fee(),
+				);
+			});
+		}
+
+		fn transfer_ausd_to_altair<T: Runtime + FudgeSupport>() {
```

Comment:
This test should best be triggered from the relay to altair. And not sibling to altair.

Reply:
EDIT: 

This is actually a good test. We should though think about having one where ausd acts as the reserve.